packages/react/src/user/user.styles.ts

```diff
@@ -38,7 +38,7 @@ export const StyledUserDesc = styled('span', {
   }
 });
 
-export const StyledUserLink = styled('a', {
+export const StyledUserLink = styled('span', {
   a: {
```

Comment:
If you change the tag into `span`, the style should also be changed together.

Reply:
It' s already styled in the [`Link`](https://github.com/nextui-org/nextui/blob/3aa92a9e0b63a06821bded15c063620f46fc4535/packages/react/src/link/link.tsx#L26) component.

Hi @jrgarciadev We can remove the `StyledUserLink` and use `Link` component directly in the `UserLink` component, since the hover style already in [`StyledLink`](https://github.com/nextui-org/nextui/blob/3aa92a9e0b63a06821bded15c063620f46fc4535/packages/react/src/link/link.styles.ts#L11) and also get a cleaner dom, I can update it if it sounds good.

Reply:
Hey @tianenpang yes, of course, it sounds good

Reply:
Done.